docs/.vuepress/config.js

```diff
@@ -55,11 +55,14 @@ module.exports = {
     docsDir: "docs",
     editLinks: false,
     sidebarDepth: 0,
-    search: true,
+    // TODO: Reenable as soon as the doc search index is good with meta
+    search: false,
     searchMaxSuggestions: 10,
     algolia: {
-      apiKey: "",
-      indexName: ""
+      apiKey: "4224b11bee5bf294f73032a4988a00ea",
```

Comment:
I suppose this is a properly restricted key that can be made public

Reply:
Yes that's the instructions they gave me in their intro email:

JavaScript
Copy this snippet at the end of the HTML <body> tag
<script src="https://cdn.jsdelivr.net/npm/@docsearch/js@3"></script>
 <script type="text/javascript">
 docsearch({
 appId: RSEEOBCB49,
 apiKey: 4224b11bee5bf294f73032a4988a00ea,
 indexName: kuma,
 container: '### REPLACE ME WITH A CONTAINER (e.g. div) ###'
 debug: false // Set debug to true if you want to inspect the modal
 });
 </script>